perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -1645,22 +2092,33 @@ public Object mapRow(ResultSet rs, int i) throws SQLException {
 
 	@Override
 	public String getEntitylessAttrValueForUpdate(PerunSession sess, int attrId, String key) throws InternalErrorException, AttributeNotExistsException {
```

Comment:
I am little afraid of this one. This specific one using "SELECT FOR UPDATE" in SQL, not sure how it should work, if both cache and DB are working together. We should think about this one.